deploy/roles/odbc/templates/odbcinst.ini

```diff
@@ -30,7 +30,7 @@ useNTLMv2               = True
 
 {% for driver in odbcinst_entries %}
 [{{ driver.drivername }}]
-{% for key, value in driver.attributes.iteritems() %}
+{% for key, value in driver.attributes.items() %}
```

Comment:
{% for key, value in list( driver.attributes.items() ) %}

if you want it to work in both 2 and 3